File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/bank/presenter/UpiPinPresenter.java
package org.mifos.mobilewallet.mifospay.bank.presenter;

import org.mifos.mobilewallet.mifospay.bank.BankContract;
import org.mifos.mobilewallet.mifospay.base.BaseView;

import javax.inject.Inject;

/**
 * Created by ankur on 13/July/2018
 */

public class UpiPinPresenter implements BankContract.UpiPinPresenter {

    BankContract.UpiPinView mUpiPinView;

    @Inject
    public UpiPinPresenter() {
    }

    @Override
    public void attachView(BaseView baseView) {
        mUpiPinView = (BankContract.UpiPinView) baseView;
    }
}

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/bank/presenter/UpiPinPresenter.kt
package org.mifos.mobilewallet.mifospay.bank.presenter

import org.mifos.mobilewallet.mifospay.bank.BankContract
import org.mifos.mobilewallet.mifospay.bank.BankContract.UpiPinView
import org.mifos.mobilewallet.mifospay.base.BaseView
import javax.inject.Inject

/**
 * Created by ankur on 13/July/2018
 */
class UpiPinPresenter @Inject constructor() : BankContract.UpiPinPresenter {
    var mUpiPinView: UpiPinView? = null
    override fun attachView(baseView: BaseView<*>?) {
        mUpiPinView = baseView as UpiPinView?
    }
}